docs/src/main/sphinx/release/release-452.md

```diff
@@ -0,0 +1,47 @@
+# Release 452 (3 Jul 2024)
+
+## General
+
+* Fix query plans occasionally not rendering the stage details page. ({issue}`22542`)
+* Fix a performance regression when using the native filesystem for Azure. ({issue}`22561`)
```

Comment:
Dont we do that in the connectors normally?

Reply:
What connectors would this go to? It's just a filesystem, yeah?

Reply:
Lake Connectors.